modules/logging-release-notes-5-6-18.adoc

```diff
@@ -0,0 +1,32 @@
+// module included in /logging/logging-5-6-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-6-18_{context}"]
+= Logging 5.6.18
+This release includes link:https://access.redhat.com/errata/RHSA-2024:2092[OpenShift Logging Bug Fix 5.6.18]
```

Comment:
Should there be a date for the release? This link doesn't work but I assume it will on the day of release. But, I don't know when that is.

Reply:
The link will work fine only when the errata is live. It's live now so it will work. :)